hadoop-hdds/rocksdb-checkpoint-differ/src/main/java/org/apache/ozone/rocksdiff/RocksDBCheckpointDiffer.java

```diff
@@ -541,7 +541,7 @@ public void onCompactionCompleted(RocksDB db,
           // TODO: [SNAPSHOT] Once SnapshotChainManager is put into use,
           //  set snapshotID to snapshotChainManager.getLatestGlobalSnapshot()
```

Comment:
Can you please remove this comment as well?

Reply:
done, thanks!